packages/integration-tests/index.js

```diff
@@ -142,7 +142,7 @@ async function getExecutionStatus(executionArn) {
     const { status } = await StepFunctions.describeExecution({ executionArn });
     return status;
   } catch (error) {
-    if (error.code === 'ExecutionDoesNotExist') return 'STARTING';
+    if (error instanceof StepFunctions.ExecutionDoesNotExist) return 'STARTING';
```

Comment:
Trying to keep changes to a minimum and access these errors via existing imports. Happy to adjust if anyone feels strongly and e.g. import `ExecutionDoesNotExist` explicitly.

Reply:
Yeah, I do agree that it's better to only import what we need. I started to make the change but found why we're doing it this way, at least in some cases.

In some tests we're importing the entire `StepFunctions` module because we need to stub methods on that module. We can't do it exactly the same way if we import a member of the module. e.g.

```
sinon.stub(StepFunctions, 'getExecutionStatus').callsFake(...)
```

won't work if we import `getExecutionStatus` directly. 

That makes me more inclined to not mess with the imports as there are (again, in some cases) functional issues doing it one way vs. the other. I'm not sure the change is worth the trouble 🤔. Thoughts?